API.md

```diff
@@ -2,8 +2,7 @@
 
 ## Constructor
 ```c
-BLEPeripheral(unsigned char req, unsigned char rdy, unsigned char rst);
-```
+BLEPeripheral(unsigned char req = BLE_DEFAULT_REQ, unsigned char rdy = BLE_DEFAULT_RDY, unsigned char rst = BLE_DEFAULT_RST);
  * req - REQ pin
  * rdy - RDY pin
  * rst - RST pin, can be ```UNUSED```
```

Comment:
Right, probably need to refer to https://github.com/sandeepmistry/arduino-BLEPeripheral#pinouts

so just a link?